ansible/roles/networking/tasks/main.yaml

```diff
@@ -36,6 +36,16 @@
       state: present
       reload: yes
 
+  - name: Flush Iptable rules
+    iptables:
+      table: "{{ item }}"
+      flush: true
+    with_items:
+      - nat
+      - raw
+    when:
+      - flush_iptables_rules | default(false)
```

Comment:
Should this also be `default(true)` as it was in konvoy 1

Reply:
No that was on reset, not the initial install, we can keep it `false`.